### ISSUE TITLE
Error in Chrome after the update to v15

### ISSUE THREAD

**Issue by Lesko987** [wikitree/wikitree-browser-extension#115]
After updating to v15 I got an error on extension reload. With dist folder and with published zip file.

After Inspecting view in service worker I found this error

Error in event handler: ReferenceError: window is not defined
    at chrome-extension://aicngmhgodhjdeiblaikecebkoaaoaac/background.js:7:5

And clicking the code link returns 

chrome.runtime.onInstalled.addListener(function (details) {
  if (details.reason == "install") {
    chrome.runtime.openOptionsPage();
    window.open("https://www.wikitree.com/wiki/Space:WikiTree_Browser_Extension%22);
  } else if (details.reason == "update") {
    // Remove the slashes below after a significant update (e.g. a new feature).
    window.open("https://www.wikitree.com/wiki/Space:WikiTree_Browser_Extension_Update%22);
  }
});

I think the error was on 2nd window.open command. The extension otherwise works.

**Comment by RobPavey** [comment]
I just started getting this too. It probably started for us when a background.js was added to our extension.

It is a known bug in Chrome:
https://bugs.chromium.org/p/chromium/issues/detail?id=1257170

There is a workaround described on that page:
1. if you already clicked Errors, close this tab as it's unusable now
2. open chrome://extensions
3. open devtools console
4. paste this in console and press Enter key:
   URL=class extends URL { constructor(href, ...rest) { super(href || 'dummy://', ...rest) } }
5. click Errors in your extension's card
